CONTRIBUTING.md

```diff
@@ -7,12 +7,47 @@ If you have any questions about how to contribute, either [create a GH issue](ht
 
 ## Developing KSQL
 
+### About the Apache Maven wrapper
+
+Development versions of KSQL use version ranges for dependencies
+on other Confluent projects, and due to
+[a bug in Apache Maven](https://issues.apache.org/jira/browse/MRESOLVER-164),
+you may find that both Maven and your IDE download hundreds or thousands
+of pom files while resolving dependencies. They get cached, so it will
+be most apparent on fresh forks or switching to branches that you haven't
+used in a while.
+
+Until we are able to get a fix in and released officially, we need to work
+around the bug. We have added a Maven wrapper script and configured
+it to download a patched version of Maven. You can use this wrapper simply by
+substituting `./mvnw` instead of `mvn` for every Maven invocation.
+The patch is experimental, so if it causes you some trouble, you can switch
+back to the official release just by using `mvn` again. Please let us know
+if you do have any trouble with the wrapper.
+
+You can also configure IntelliJ IDEA to use the patched version of Maven.
```

Comment:
This will probably be welcome news for devs. Hopefully, it works as expected! I have been having a couple of odd problems today, but I think it's just because `master` is somewhat broken right now.

Reply:
Nice~